src/main/surfrdMod.F90

```diff
@@ -650,6 +618,37 @@ subroutine surfrd_special(begg, endg, ncid, ns)
 
     deallocate(pctgla,pctlak,pctwet,pcturb,pcturb_tot,urban_region_id,pctspec)
 
+    ! Select N dominant landunits
+    ! ---------------------------
+    ! n_dom_landunits set by user in namelist
+    ! Call resembles the surfrd_veg_all call collapse_nat_pfts that selects
+    ! n_dom_pfts (also set by the user in the namelist)
+    !
+    ! Development steps:
+    ! 1. Rank special landunits only, without affecting the vegetated landunits
+    ! 2. When (1) works, continue: Rank the vegetated landunits as one value
+    !    against the special landunits. If the vegetated landunits get rejected
+    !    as not dominant, this information will need to transmit to
+    !    surfrd_veg_all
```

Comment:
I was drawn to calling this new subroutine from here because we're in surfrd_special; however, at this stage in the run we do not have wt_lunit for istsoil and iscrop, so I'm starting to rethink the position of the new call.

Reply:
@slevisconsulting I think this question was resolved. Just asking you to check for sure.

Reply:
Yes, this code is outdated and that comment has been removed.